bundles/binding/org.openhab.binding.fht/src/main/java/org/openhab/binding/fht/internal/FHTBinding.java

```diff
@@ -270,6 +239,7 @@ protected void removeBindingProvider(FHTBindingProvider bindingProvider) {
      */
     @Override
     public void updated(Dictionary<String, ?> config) throws ConfigurationException {
+        setProperlyConfigured(false);
```

Comment:
I haven't looked if it's true for AbstractBindings, but for AbstractActiveBindings it has side effects to `setProperlyConfigured(false)` and then `setProperlyConfigured(true)`.  A different pattern could be:
```java
if (config != null) {
    boolean properlyConfigured = false;
    try {
        // parse and throw as needed...
        properlyConfigured = true;
    } finally {
        setProperlyConfigured(properlyConfigured);
    }
}
```